insights/core/__init__.py

```diff
@@ -1663,6 +1663,65 @@ def path_entry(self, path):
         return self.listings[directory]['entries'][name]
 
 
+class DiskUsage(CommandParser, dict):
+    """
+    Reads output of du command and turns it into a dictionary with pathname as
+    key and size in blocks.
+
+    Supports parsing input data as long as output is 2 column data with first
+    column as space size with integer values only and second as pathname which
+    can be a file or directory. Space size with decimal values or unit suffixes
+    like M, GiB is not supported.
+
+    du command produces output in 1K blocks unless block size is specified in
+    command options or an environment variable. This parser is intended to be
+    used only with default block size of 1K which is also equal to plain "du"
+    or "du -k".
+
+    Sample input data:
```

Comment:
Please include `::` at the end of this line to correctly format the sample input below.  You can review the rendered documentation following the instructions in the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#testing-your-docstring).

Reply:
Done.